src/collection/components/CollectionOrBundleEditMetaData.tsx

```diff
@@ -53,17 +52,51 @@ const CollectionOrBundleEditMetaData: FunctionComponent<CollectionOrBundleEditMe
 	const [descriptionLongEditorState, setDescriptionLongEditorState] = useState<
 		RichEditorState | undefined
 	>(undefined);
+	const [lomFields, setLomFields] = useState<LomFieldsByScheme>({
+		contexts: [],
+		educationLevels: [],
+		subjects: [],
+		themes: [],
+	});
 
 	const isCollection = type === 'collection';
 
-	const updateCollectionMultiProperty = (
-		selectedTagOptions: TagInfo[],
-		collectionProp: keyof Avo.Collection.Collection
-	) => {
+	useEffect(() => {
+		if (!isEmpty(collection) && collection.loms) {
+			const groupedLoms = groupLoms(collection.loms);
+
+			setLomFields({
+				...lomFields,
+				educationLevels:
+					mapLomsToLomFields(
+						groupedLoms[LomSchemeTypeEnum.structure] as Avo.Lom.LomEntry[]
+					) || [],
+				subjects:
+					mapLomsToLomFields(
+						groupedLoms[LomSchemeTypeEnum.subject] as Avo.Lom.LomEntry[]
+					) || [],
+				themes:
+					mapLomsToLomFields(
+						groupedLoms[LomSchemeTypeEnum.theme] as Avo.Lom.LomEntry[]
+					) || [],
+			});
```

Comment:
saving these fields in a local state usually causes strange issues where this state doesn't stay in sync with the parent lom fields state

Its better to just make this a computed value:
```typescript
const lomFields = useMemo(() => {
	const groupedLoms = groupLoms(collection.loms);
	return {
		educationLevels: mapLomsToLomFields(
			groupedLoms[LomSchemeTypeEnum.structure] as Avo.Lom.LomEntry[]
		) || [],
		subjects: mapLomsToLomFields(
			groupedLoms[LomSchemeTypeEnum.subject] as Avo.Lom.LomEntry[]
		) || [],
		themes: mapLomsToLomFields(
			groupedLoms[LomSchemeTypeEnum.theme] as Avo.Lom.LomEntry[]
		) || [],
	}
}, [collection.loms]);
```

![IMG_20230607_121424 (1)](https://github.com/viaacode/avo2-client/assets/1710840/53245197-6e82-47c8-9d00-b901eac8ca0d)